### PR TITLE
Update Elasticsearch

### PR DESCRIPTION
**Pull request by bensteinberg**
https://www.elastic.co/guide/en/elasticsearch/reference/7.17/release-notes-7.17.17.html

### CODE REVIEW

**Comment by bensteinberg** [comment]
@teovin, for context, when I see that there's a package update for a package specified in our codebase, I submit a PR like this and wait until it's merged to perform the update. This isn't critical.

**Review by teovin — APPROVED** [review]
Thank you for the context, @bensteinberg!

**Pull request merged**
